docs/components/emails.rst

```diff
@@ -9,25 +9,83 @@ There are multiple ways to extend the way Mautic works with Emails. This documen
 - Email transport/Email providers
 - Email stat helpers
 
-Email tokens and A/B testing
-----------------------------
+.. Note:: Extending generally works by hooking into events using event listeners or subscribers. Read more about :doc:`listeners and subscribers</plugins/event_listeners>`.
+
+Email tokens
+------------
 
 Email tokens are placeholders that you can insert into an Email.
 They get replaced by Dynamic Content once the Email gets sent or viewed in the browser.
 
-You can find examples of both Email token handling and A/B testing in the code example below.
-Both leverage the ``\Mautic\EmailBundle\EmailEvents::EMAIL_ON_BUILD`` event. Read more about :doc:`listeners and subscribers</plugins/event_listeners>`.
-
 Email token capabilities consist of two parts: registering your custom tokens and rendering them.
 
-- ``$event->addToken($uniqueId, $htmlContent)`` allows you to show the Email token in the email builder, so that users can easily add the token to their emails.
-- ``$event->getContent()`` and ``$event->setContent()`` are used for replacing the Email token with actual Dynamic Content once the Email gets send or viewed in the browser.
+Registering custom tokens in builders
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
We use the following:

H1 ===
H2 ***
H3 ---
H4 ---
H5 ~~~
H6 ^^^

I think these should therefore be using ~~~?

Reply:
Changed